simpleflow/swf/process/actor.py

```diff
@@ -10,13 +10,14 @@
 import swf.actors
 import swf.exceptions
 from simpleflow import utils
+from simpleflow.process import NamedMixin, with_state
 from simpleflow.swf.helpers import swf_identity
 
 
 logger = logging.getLogger(__name__)
 
 
-__all__ = ['Supervisor', 'Poller']
+__all__ = ['Poller']
```

Comment:
I saw it has some effects if you `from foo import *`, it only imports the given names and not all the names exposed in the module (which can include 2nd level deps..). Am I mistaken?

I only maintained it because it was here in the first time btw, we can remove it...